msws.sh

```diff
@@ -1,16 +1,21 @@
+state_file="$HOME/.last_msws"
```

Comment:
I'm running this as a systemd service so will probably want to store this in the service's $STATE_DIRECTORY, may be better to execute this as a generic environment variable for flexibility.